feature-libs/cart/import-export/styles/_import-entries-dialog.scss

```diff
@@ -1,14 +1,11 @@
 cx-import-entries-dialog {
   background-color: rgba(0, 0, 0, 0.5);
-
   .cx-import-entries-dialog {
     @extend .modal-dialog;
     @extend .modal-dialog-centered;
 
     .cx-import-entries-container {
       @extend .modal-content;
-      pointer-events: none;
```

Comment:
I can see that the ability to turn off the modal by clicking the mouse outside of it is associated with the removal of another functionality (equally important, maybe even more important) - selecting and copying text/output.

In this case, we especially need consent to the changes @Xymmer

Reply:
Update: I can see in add to cart dialog, we are able to both: select text and close by clicking outside. Can't you do the same here?

Reply:
Adding  "tabindex="0" to the cx-import-entries-container lets the user select text